Bootstrap.php

```diff
@@ -12,8 +12,11 @@
 
 
 namespace DataTables;
-if (!defined('DATATABLES')) exit();
 
+// Ensure Editor is loadable.
+if (!defined('DATATABLES') && !class_exists('\DataTables\Editor')) {
```

Comment:
@AllanJard I have updated this PR, now we check if `DATATABLES` const is defined - for BC and usage /wo composer.

With composer, the second check will do the work and no global `DATATABLES` const is even needed.